{"uuid": "82f2d123-5a9c-4f0c-a666-c3002158fee1", "children": ["79768048-e922-4dab-ac7d-a5bf5e088570", "0bec6509-6ca9-4dcb-bec0-a9da76a435fb"], "befores": [{"name": "setup_profiles", "status": "broken", "statusDetails": {"message": "KeyError: 'ifname'\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/wlan-testing/tests/e2e/basic/conftest.py\", line 80, in setup_profiles\n    return_var = fixtures_ver.setup_profiles(request, param, setup_controller, testbed, get_equipment_id,\n  File \"/wlan-testing/tests/fixtures_2x.py\", line 292, in setup_profiles\n    ap_wifi_data = ap_ssh.get_interface_details()\n  File \"/wlan-testing/tests/../libs/apnos/apnos.py\", line 403, in get_interface_details\n    wifi_info[j[\"ifname\"]] = [j[\"config\"][\"ssid\"], j[\"config\"][\"encryption\"], \"\"]\n"}, "attachments": [{"name": "ucentral_config: ", "source": "809b5768-f42c-439c-b78a-a0bb52581d0f-attachment.attach"}, {"name": "Time Took to apply Config: 139.43443274497986", "source": "ecf8bffa-7af9-420e-9cbd-c7b9748f7d35-attachment.attach"}, {"name": "Success : Active Config in AP: ", "source": "e814c288-c219-4e95-9851-06889188594b-attachment.attach"}, {"name": "AP LOgs: ", "source": "0b9261dc-d2e2-47af-9775-48b0bc2e2e2a-attachment.attach"}], "start": 1628815266095, "stop": 1628815437474}], "start": 1628815266095, "stop": 1628815437638}